app/models/keyword.rb

```diff
@@ -7,11 +7,20 @@ class Keyword < ApplicationRecord
 
   enum status: { in_progress: 0, completed: 1, failed: 2 }
 
+  has_many :links, dependent: :destroy
+
   def update_status(status)
     update(status: status)
   end
 
-  def add_html(html)
-    update(html: html, status: :completed)
+  def update_data(html:, parse_result:)
```

Comment:
Let's add test for this function

Reply:
Nice catch!
Added test in [`e6fd857` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/e6fd8573a7f7a0650c7baf4eeafd87c042c4cc4b)